quickstarts/microsoft.mobilenetwork/mobilenetwork-create-full-5gc-deployment-aks-hci/main.bicep

```diff
@@ -31,34 +31,30 @@ param simResources array = []
 @description('The platform type where packet core is deployed.')
 @allowed([
   'AKS-HCI'
-  'BaseVM'
 ])
 param platformType string = 'AKS-HCI'
 
-@description('The name of the control plane interface on the access network. In 5G networks this is called the N2 interface whereas in 4G networks this is called the S1-MME interface. This should match one of the interfaces configured on your Azure Stack Edge machine.')
+@description('The resource ID of the Azure Stack Edge device to deploy to')
+param azureStackEdgeDevice string = ''
+
+@description('The name of the control plane interface on the access network. This must match the corresponding virtual network name on port 5 on your Azure Stack Edge Pro device. For 5G, this interface is the N2 interface, whereas for 4G, it\'s the S1-MME interface.')
 param controlPlaneAccessInterfaceName string = ''
 
 @description('The IP address of the control plane interface on the access network. In 5G networks this is called the N2 interface whereas in 4G networks this is called the S1-MME interface.')
 param controlPlaneAccessIpAddress string = ''
 
-@description('The logical name of the user plane interface on the access network. In 5G networks this is called the N3 interface whereas in 4G networks this is called the S1-U interface. This should match one of the interfaces configured on your Azure Stack Edge machine.')
+@description('The name for the user plane interface on the access network. This must match the corresponding virtual network name on port 5 on your Azure Stack Edge Pro device. For 5G, this interface is the N3 interface, whereas for 4G, it\'s the S1-U interface.')
 param userPlaneAccessInterfaceName string = ''
 
-@description('The IP address of the user plane interface on the access network. In 5G networks this is called the N3 interface whereas in 4G networks this is called the S1-U interface. Not required for AKS-HCI.')
-param userPlaneAccessInterfaceIpAddress string = ''
-
 @description('The network address of the access subnet in CIDR notation')
 param accessSubnet string = ''
 
 @description('The access subnet default gateway')
 param accessGateway string = ''
 
-@description('The logical name of the user plane interface on the data network. In 5G networks this is called the N6 interface whereas in 4G networks this is called the SGi interface. This should match one of the interfaces configured on your Azure Stack Edge machine.')
+@description('The name for the user plane interface on the data network. This must match the corresponding virtual network name on port 6 on your Azure Stack Edge Pro device. For 5G, this interface is the N6 interface, whereas for 4G, it\'s the SGi interface.')
 param userPlaneDataInterfaceName string = ''
```

Comment:
See my comment in https://github.com/MicrosoftDocs/azure-docs-pr/pull/209159/files#r960985649 - I think we want to re-write these descriptions.